src/apps/competitions/tests/test_submissions.py

```diff
@@ -380,3 +382,14 @@ def test_fact_sheet_with_blank_required_text_is_not_valid(self):
         }
         serializer = SubmissionCreationSerializer(data=submission, instance=Submission)
         assert not serializer.is_valid()
+
+
+class TestSubmissionTasks(SubmissionTestCase):
+    def test_submissions_are_cancelled_if_running_24_hours_past_execution_time_limit(self):
+        self.submission = self.make_submission()
+        self.submission.status = Submission.RUNNING
+        self.submission.created_when = timezone.now() - timedelta(milliseconds=(3600000 * 24) + self.submission.phase.execution_time_limit)
```

Comment:
can you modify the test so there is a running submission that is ignored?

Reply:
Ah yeah, good idea.